src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -49,6 +54,27 @@ public function __construct($name, NodeInterface $parent = null)
         $this->parent = $parent;
     }
 
+    /**
+     * Register possible values for a dynamic placeholder value.
+     *
+     * @param string $placeholder A globally unique placeholder string
+     * @param array  $values      Possible values per primitive type
+     */
+    public static function setPlaceholder($placeholder, array $values)
+    {
+        self::$placeholders[$placeholder] = $values;
+    }
+
+    /**
+     * Reset all current placeholders available.
+     *
+     * @return void
```

Comment:
should be removed as suggested by fabbot (and the dobclokc removed altogether maybe?)